src/pipelines/vaccinations/config.yaml

```diff
@@ -226,6 +226,20 @@ sources:
       job_group: all
       deferred: true
 
+  # Data sources for SK level 0 + 1
+  - name: pipelines.vaccinations.sk_authority.SlovakiaDataSource
+    fetch:
+      - name: "regions"
+        url: "https://data.korona.gov.sk/api/regions"
+      - name: "vaccine_by_regions"
+        url: "https://data.korona.gov.sk/api/vaccinations/by-region"
+    test:
+      location_key_match:
+        - '^SK_[^_]+$'
```

Comment:
Does the data source provide country-level data? If it doesn't, you can either:
1. Aggregate it here using the `aggregate` option (see CZ for example)
2. Aggregate it in the data source code (see IT for example)

If the data source has data with unknown locations, then you have to do # 2. Otherwise # 1 is preferred since it's a no-code solution.

Reply:
It does but through a different endpoint, but the region sum matches the country sum provided by OWID, so I thought we could add it later if required.

Reply:
Then I'd suggest aggregating the regions now (just a couple of lines in the config file) and later we can add the call to the separate endpoint.

Even if the data is available through OWID, it's typically significantly delayed compared to getting it from the source directly.

Reply:
Okay - will do now.

Reply:
Done - works well!